user-data/lists/gdpr/srcs/raw.githubusercontent.com/steemit/condenser/master/src/app/utils/GDPRUserList.js

```diff
@@ -0,0 +1,7 @@
+const list = `
+mateja.klaric
+`
+    .trim()
+    .split('\n');
```

Comment:
It looks like each list has 4 derivatives, don't we need just one? Or none if they are generated in build?

Reply:
Well for gdpr, exchanges yes, there is a 1:1 match to the condenser lists, but I built it to allow for others since bad_actors includes condenser and irredeemables, and at the moment, this is updated via commit which means performance isnt an issue. I can remove the non-ts files though from the build process tough.